lib/core/i18n/i18n.js

```diff
@@ -8,6 +8,7 @@ i18n.configure({
   locales: supported_languages,
   register: global,
   updateFiles: false,
+  syncFiles: false,
```

Comment:
What does that do, just curious?

Reply:
it syncs the keys (adds new keys if any) for all files, not just english.

Reply:
when set to true